scripts/blogView.js

```diff
@@ -34,7 +34,7 @@ blogview.handleAuthorFilter = function() {
 };
```

Comment:
Can you see a way to refactor the above code, keeping in mind that the class "template" no longer exists?